packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -1112,8 +1112,22 @@ class Viewport {
       imageData.indexToWorld(idx, focalPoint);
     }
 
-    const { widthWorld, heightWorld } =
-      this._getWorldDistanceViewUpAndViewRight(bounds, viewUp, viewPlaneNormal);
+    let widthWorld;
```

Comment:
My preference is to extract this into it's own function which returns a consistent result, and use const {widthWOrld, heightWorld} = this.getSizeWorld(imageData,camera)

Reply:
I can do that